gapic/schema/wrappers.py

```diff
@@ -1200,7 +1200,8 @@ def field_headers(self) -> Sequence[FieldHeader]:
 
         http = self.options.Extensions[annotations_pb2.http]
 
-        pattern = re.compile(r'\{([a-z][\w\d_.]+)=')
+        # Copied from Node generator.
```

Comment:
Thanks! Can we add a link to the GitHub file where this regex was copied from?

Reply:
Done